include/zmq.h

```diff
@@ -557,6 +557,10 @@ ZMQ_EXPORT void zmq_threadclose (void* thread);
 
 #ifdef ZMQ_BUILD_DRAFT_API
 
+/* DRAFT Context options */
```

Comment:
There's already some below, at line 617 (ZMQ_MSG_T_SIZE), could you please fix it? Also it conflicts